src/richie-front/js/components/CourseGlimpseList/CourseGlimpseList.tsx

```diff
@@ -19,7 +19,7 @@ export class CourseGlimpseList extends React.Component<CourseGlimpseListProps> {
     const { courses } = this.props;
 
     return (
-      <div className="course-glimpse-list">
+      <div className="course-glimpses">
```

Comment:
I think it can be a "gotcha" to just add an "s" to the item name to name a list, I'd rather keep `course-glimpse-list`

Reply:
+1